drivers/ethernet/eth_stm32_hal.c

```diff
@@ -291,7 +498,7 @@ static void rx_thread(void *arg1, void *unused1, void *unused2)
 			if (dev_data->link_up != true) {
 				dev_data->link_up = true;
 				net_eth_carrier_on(get_iface(dev_data,
-							     vlan_tag));
+								   vlan_tag));
```

Comment:
original code indentation was correct, please remove the change

Reply:
done